xrmocap/core/estimation/mview_sperson_smpl_estimator.py

```diff
@@ -110,6 +110,14 @@ def __init__(self,
 
         if isinstance(smplify, dict):
             smplify['logger'] = logger
+            if smplify['type'].lower() == 'smplify':
+                self.smpl_data_type = 'smpl'
+            elif smplify['type'].lower() == 'smplifyx':
+                self.smpl_data_type = 'smplx'
+            else:
+                self.logger.warning('smpl data type not defined, '
```

Comment:
If `smplify['type']` does not hit either of them, it is impossible to build_registrant. What about `self.logger.error('SMPL data type not found.')` and raise TypeError?

Reply:
Fixed in [f33d0ca](https://github.com/openxrlab/xrmocap/pull/52/commits/f33d0caed97adb0a3c2afe553b5f9f34503f900f).

Reply:
`logger.error` does not stop the program. To prevent it from going on, `raise xxxError` is necessary. Otherwise, a user has to read a lot of building failure messages in console, not the log message we provide.

Reply:
Fixed in [21a9b15](https://github.com/openxrlab/xrmocap/pull/52/commits/21a9b1598c19c0618ff463203501f7afeea2e43d).